backend/qemu.pm

```diff
@@ -490,6 +490,7 @@ sub start_qemu {
     bmwqemu::save_vars();                 # update variables
 
     mkpath($basedir);
+    runcmd('/usr/bin/chattr', '+C', $basedir);
```

Comment:
Unfortunately, the 'sys/ioctl.ph' is quite broken:
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=190887

```
#! /usr/bin/perl -w

require "linux/fs.ph";

printf "0x%x\n", FS_IOC_GETFLAGS();
printf "0x%x\n", FS_IOC_SETFLAGS();

$sizeof{'long'} = 8;
print eval "$sizeof{'long'}", "\n";
printf "0x%x\n", FS_IOC_GETFLAGS();
printf "0x%x\n", FS_IOC_SETFLAGS();
```
returns: 
```
Use of uninitialized value $size in left bitshift (<<) at (eval 221) line 1.
0x80006601
Use of uninitialized value $size in left bitshift (<<) at (eval 224) line 1.
0x40006602
8
0x80086601
0x40086602
```